plugins/services/src/js/components/forms/EnvironmentFormSection.js

```diff
@@ -1,4 +1,5 @@
 import React, {Component} from 'react';
+import {MountService} from 'foundation-ui';
```

Comment:
Hm... Let's unblock this PR and leave it as it is for now. We can revisit this within the next package/plugin iteration.